### PR TITLE
Changed the images library back to sharp

### PR DESCRIPTION
**Pull request by chenlevy24**
now bubblewrap works with svg images

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,5 +33,8 @@
     "lerna-changelog": "^1.0.1",
     "npm": "^6.14.5",
     "typescript": "^3.9.2"
+  },
+  "dependencies": {
```
Comment (andreban) [inline package.json:37]: We don't need sharp as a dependency for the root `package.json`. We should remove it here.

File: packages/core/package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     "color": "^3.1.2",
     "extract-zip": "^1.7.0",
     "inquirer": "^7.0.4",
-    "jimp": "^0.9.8",
+    "sharp": "^0.23.2",
```
Comment (andreban) [inline packages/core/package.json:44]: We should also remove the dependency on `jimp` from the `packages/core/package.json`

File: packages/core/src/lib/TwaGenerator.ts [hunk 3]
```diff
@@ -20,8 +20,8 @@ import fetch from 'node-fetch';
 import {template} from 'lodash';
 import {promisify} from 'util';
 import {TwaManifest, ShortcutInfo} from './TwaManifest';
-import Jimp = require('jimp');
 import Log from './Log';
+const sharp = require('sharp');
```
Comment (andreban) [inline packages/core/src/lib/TwaGenerator.ts:24]: Sharp has Typescript types: https://www.npmjs.com/package/@types/sharp. We can install it with `npm install --save @types/sharp` in the packages/core folder.

The, we can use a proper import statement here (and also get type checking in the code)

File: packages/core/src/lib/TwaGenerator.ts [hunk 4]
```diff
@@ -142,9 +142,12 @@ export class TwaGenerator {
   }
 
   private async saveIcon(data: Buffer, size: number, fileName: string): Promise<void> {
-    const image = await Jimp.read(data);
-    await image.resize(size, size);
-    await image.writeAsync(fileName);
+    console.log("ss");
```
Comment (andreban) [inline packages/core/src/lib/TwaGenerator.ts:145]: Remove the dangling `console.log`

File: packages/core/src/lib/TwaGenerator.ts [hunk 5]
```diff
@@ -142,9 +142,12 @@ export class TwaGenerator {
...
+    const image =await sharp(data)
```
Comment (andreban) [inline packages/core/src/lib/TwaGenerator.ts:146]: nit: missing space after the `=` sign.

File: packages/core/src/lib/TwaGenerator.ts [hunk 6]
```diff
@@ -142,9 +142,12 @@ export class TwaGenerator {
...
+    .resize(size)
```
Comment (andreban) [inline packages/core/src/lib/TwaGenerator.ts:147]: nit: ident the lines below.

File: packages/core/src/lib/TwaGenerator.ts [hunk 7]
```diff
@@ -16,12 +16,13 @@
...
 import * as path from 'path';
 import * as fs from 'fs';
+import * as sharp from 'sharp';
 import fetch from 'node-fetch';
 import {template} from 'lodash';
 import {promisify} from 'util';
 import {TwaManifest, ShortcutInfo} from './TwaManifest';
 import Log from './Log';
-const sharp = require('sharp');
+
```
Comment (PEConn) [inline packages/core/src/lib/TwaGenerator.ts:25]: nit: Remove empty line.

**Pull request merged**